parsons/etl/tofrom.py

```diff
@@ -771,4 +771,4 @@ def from_dataframe(cls, dataframe, include_index=False):
                 Include index column
         """
 
-        return cls(petl.fromdataframe(dataframe, include_index=include_index))
+        return cls(petl.fromdataframe(dataframe, include_index=include_index))
```

Comment:
This change here is what's causing the tests to fail. It's a [linting](https://dbader.org/blog/python-code-linting) issue. That's fine, because we need to remove this file from the PR anyway. I should be able to do that once your PR is otherwise ready to go.